src/features/auth/auth.service.ts

```diff
@@ -32,6 +32,7 @@ export class AuthService {
       throw new WrongPasswordException();
     }
 
+    (session as any).news = '';
```

Comment:
It's interesting that I need to interact with the session object somehow. If I don't, cookies aren't returned to clients. I've explored various resources but haven't been able to find a solution

Reply:
that is fine for the pet project. I am more interested in knowing that you understand the process. That is the most important factor